virtool/jobs/api.py

```diff
@@ -262,4 +262,4 @@ async def push_status(req):
     except ResourceConflictError:
         raise HTTPConflict(text="Job is finished")
 
-    return json_response(document["status"][-1], status=201)
+    return json_response(document, status=201)
```

Comment:
You had this right before. You are creating a new status resource at this endpoint and should return it as the response, not the parent job.

Reply:
I created and returned the status in the data layer, which is why I just return document here. Is this incorrect?

Reply:
Sorry, that part was hidden in the review tool. You're right.

Reply:
You're right!